frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -143,91 +147,123 @@ const Uploads = () => {
     <main className="prime-home">
       <div className="grid-container">
         <div className="grid-row">
-          <div className="prime-container">
+          <div className="prime-container card-container">
             <div className="usa-card__header">
-              <h2>COVID-19 CSV Uploads</h2>
+              <h2>Upload your results</h2>
             </div>
-            {reportId && (
-              <div>
-                <div className="usa-alert usa-alert--success">
-                  <div className="usa-alert__body">
-                    <h4 className="usa-alert__heading">
-                      Success: File Accepted
-                    </h4>
-                    <p className="usa-alert__text">
-                      Your file has been successfully transmitted to the
-                      department of health
+            <div className="usa-card__body">
+              <p>
+                Add and report results in bulk using a comma-separated values
+                (CSV) spreadsheet. To upload your spreadsheet:
+              </p>
+              <ul className="usa-list">
+                <li>
+                  <a href="https://reportstream.cdc.gov/assets/csv/ReportStream-StandardCSV-ExampleData-20220509.csv">
+                    Download the spreadsheet template
+                  </a>
+                </li>
+                <li>
+                  Add your data to the relevant fields in the template, or if
+                  you’re using your own spreadsheet, format column headers to
+                  match the template
+                </li>
+                <li>
+                  Make sure you include data for all fields marked “required” in
+                  row 2
+                </li>
+                <li>Once you’ve entered your data, delete all example data</li>
+                <li>Save your spreadsheet as a CSV file and upload it here</li>
+              </ul>
+              <p>
+                For more information about the spreadsheet, visit our{" "}
+                <LinkWithQuery to="/results/upload/schema">
+                  data template guide.
+                </LinkWithQuery>
```

Comment:
```suggestion
                For more information about this process, visit our{" "}
                <LinkWithQuery to="/results/upload/schema">
                  data template guide
                </LinkWithQuery> or upload guide.
```
Will need to add link to "upload guide," but not totally sure how to

Reply:
Done in [afb0b7d](https://github.com/CDCgov/prime-simplereport/pull/3928/commits/afb0b7daff592ea23efc832618b88235dd03cac0) 👍 the upload guide is the `CsvUploadGuide.tsx` file that you reviewed as a part of this PR so we should be good